app/[locale]/institute/sections/central-library/library-committee/page.tsx

```diff
@@ -0,0 +1,137 @@
+import { Suspense } from 'react';
+
+import Heading from '~/components/heading';
+import Loading from '~/components/loading';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+
+export default async function libraryCommittee({
+  params: { locale },
+}: {
+  params: { locale: string };
+}) {
+  const libraryCommitteeData = [
```

Comment:
Surely this would go into the db?

Reply:
yes, but we need to update the schema accordingly, will discuss with aryan. For now, a static object is fine?

Reply:
No